pages/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { FormEvent, HTMLAttributes, useEffect, useState } from 'react';
+import React, { FormEvent, HTMLAttributes, useEffect, useState } from 'react';
```

Comment:
Lisäsin `React`in importteihin. Kun käytti seuraavaa elementtiä: `<> </>`, tuli seuraava varoitus:
`'React' refers to a UMD global, but the current file is a module. Consider adding an import instead`

Reactin importtauksella katosi tuo varoitus

Reply:
Hetkinen, tuo onkin joku "uusi" ongelma. 

https://stackoverflow.com/a/79047801

VSCode:n "JavaScript and TypeScript Nightly" -pluginiin oli tullut update, vaati tuon kyseisen pluginin uudelleenkäynnistämisen. Nyt tuo virhe katosi vaikka poisti `React`-importin

Reply:
Poistin `React`-importit. Ne kaikki liittyi tuohon samaan ongelmaan